Automattic/woocommerce-payments
# Pass currency and do not pass transaction_ids parameter to create instant deposit

Fixes https://github.com/Automattic/woocommerce-payments/issues/7711

#### Changes proposed in this Pull Request

When creating instant deposit:
- Pass `currency` parameter.
- Stop sending `transaction_ids` parameter.

#### Testing instructions

* [Be eligible for instant deposit](https://woo.com/document/woopayments/deposits/instant-deposits/#eligibility-requirements).
* Purchase a product so balance is non-zero.
* **Use server branch from server PR 4061.**
* Open Payments > Overview page.
* Confirm 'Deposit available funds' button shows up.
* Click that button will bring up a modal.
* Open browser's dev tool and be ready to see the parameters of POST request to `/wp-json/wc/v3/payments/deposits`.
* Click the main 'Deposit $xx now' button.
* Confirm that the POST request to `/wp-json/wc/v3/payments/deposits` has `currency` parameter and not `transaction_ids`.
* Confirm that instant deposit is created successfully by going to Payments > Deposits > click on the newly created deposit. Confirm that it's an 'instant' deposit.

![Screenshot 2023-12-05 at 05 21 38](https://github.com/Automattic/woocommerce-payments/assets/73803630/5fdcdfbc-9f07-4d44-b938-e7c8b9819f58)


ps: after a successful instant deposit, the 'Deposit available funds' button state does not get updated, ie if user has exceed daily limit, the button should be hidden, etc. I'll create a separate issue for this.

-------------------


**Post merge**
